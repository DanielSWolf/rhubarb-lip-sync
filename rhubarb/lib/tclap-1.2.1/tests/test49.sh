#!/bin/sh

# failure
../examples/test8  -s bbb homer marge bart -- -hv two > tmp.out 2>&1

if cmp -s tmp.out $srcdir/test49.out; then
	exit 0
else
	exit 1
fi

