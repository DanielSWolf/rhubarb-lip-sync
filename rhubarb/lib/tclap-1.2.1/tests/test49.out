PARSE ERROR: Argument: -s (--stringTest)
             Couldn't find delimiter for this argument!

Brief USAGE: 
   ../examples/test8  [-f=<float>] ...  [-i=<int>] ...  -s=<string> [-B]
                      [--] [--version] [-h] <string> <string>
                      <fileNameString> ...

For complete USAGE and HELP type: 
   ../examples/test8 --help

